From 1a0db3e329966c1187177567935e82dc8901fe85 Mon Sep 17 00:00:00 2001
From: saikumaru <saikumaru@ami.com>
Date: Fri, 19 Sep 2025 10:15:56 +0530
Subject: [PATCH] hw/i2c:nuvoton: log bus timeout and retries

This commit adds code to log the bus timeout and retry values used
per bus. This is helpful to confirm what values are actually used
on each boot.

Signed-off-by: saikumaru <saikumaru@ami.com>
---
 drivers/i2c/busses/i2c-npcm7xx.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/i2c/busses/i2c-npcm7xx.c b/drivers/i2c/busses/i2c-npcm7xx.c
index 57c8425d4d92..559ce555e474 100644
--- a/drivers/i2c/busses/i2c-npcm7xx.c
+++ b/drivers/i2c/busses/i2c-npcm7xx.c
@@ -2449,6 +2449,8 @@ static int npcm_i2c_probe_bus(struct platform_device *pdev)
 				   "bus-retries", &bus_retries);
 	adap->retries = (ret < 0) ? 3 : bus_retries;
 
+	dev_info(bus->dev, "i2c bus %d timeout=%d, retries=%d\n",
+		 bus->num, adap->timeout, adap->retries);
 
 	irq = platform_get_irq(pdev, 0);
 	if (irq < 0)
-- 
2.47.1

